es-vue-base/src/lib-components/EsReview.vue

```diff
@@ -183,6 +309,14 @@ export default {
             // Displays DD/MM/YYYY
             return new Date(this.created).toLocaleDateString();
         },
+        updatedDate() {
+            // Displays DD/MM/YYYY
+            return new Date(this.modified).toLocaleDateString();
+        },
+        responseDate() {
+            // Displays DD/MM/YYYY
+            return new Date(this.response.modified).toLocaleDateString();
+        },
```

Comment:
This could be turned into a method to reduce repeating code.

```
<div>
{{ localDate(modified) }}
</div>


localDate(date) {
    return new Date(date).toLocalDateString();
}
```